From 5ac5c362ad56b384af6ca753667dc666994b2c3c Mon Sep 17 00:00:00 2001
From: Geert Uytterhoeven <geert@linux-m68k.org>
Date: Sun, 29 Mar 2009 11:14:59 +0200
Subject: [PATCH] zorro: use dev_name() instead of obsolete device.bus_id

device.bus_id got removed in commit 1fa5ae857bb14f6046205171d98506d8112dd74e
("driver core: get rid of struct device's bus_id string array"), so use
dev_name() instead.

Signed-off-by: Geert Uytterhoeven <geert@linux-m68k.org>
---
 drivers/zorro/zorro-driver.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/drivers/zorro/zorro-driver.c b/drivers/zorro/zorro-driver.c
index aeb0615..409244b 100644
--- a/drivers/zorro/zorro-driver.c
+++ b/drivers/zorro/zorro-driver.c
@@ -150,7 +150,7 @@ static int zorro_uevent(struct device *dev, struct kobj_uevent_env *env)
 		return -ENODEV;
 
 	if (add_uevent_var(env, "ZORRO_ID=%08X", z->id) ||
-	    add_uevent_var(env, "ZORRO_SLOT_NAME=%s", z->dev.bus_id) ||
+	    add_uevent_var(env, "ZORRO_SLOT_NAME=%s", dev_name(dev)) ||
 	    add_uevent_var(env, "ZORRO_SLOT_ADDR=%04X", z->slotaddr) ||
 	    add_uevent_var(env, "MODALIAS=" ZORRO_DEVICE_MODALIAS_FMT, z->id))
 		return -ENOMEM;
-- 
1.5.6.5

